experimental/dataloader-cifar10/Main.hs

```diff
@@ -1,46 +1,48 @@
-{-# LANGUAGE BlockArguments #-}
-{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE BlockArguments        #-}
+{-# LANGUAGE DeriveGeneric         #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
-{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE RecordWildCards       #-}
 
 module Main where
 
 import Control.Monad (when)
 import GHC.Generics
+import Prelude       hiding (exp)
 import System.Random (mkStdGen, randoms)
-import Prelude hiding (exp)
 
-import Torch
-import Torch.Vision
-import Numeric.Dataloader
-import Numeric.Datasets
-import Numeric.Datasets.CIFAR10
-import qualified Streaming as S
-import qualified Streaming.Prelude as S
-import qualified Codec.Picture as I
+import qualified Codec.Picture            as I
+import           Numeric.Dataloader
+import           Numeric.Datasets
+import           Numeric.Datasets.CIFAR10
+import qualified Streaming                as S
+import qualified Streaming.Prelude        as S
+import           Torch
+import           Torch.Vision
 
-data MLPSpec = MLPSpec {
-    inputFeatures :: Int,
-    hiddenFeatures0 :: Int,
-    hiddenFeatures1 :: Int,
-    outputFeatures :: Int
-    } deriving (Show, Eq)
+data MLPSpec = MLPSpec
+    { inputFeatures :: Int
+    , hiddenFeatures0 :: Int
```

Comment:
sometimes it moves the comma to the front (like here), sometimes it doesn't (like in examples/vae/Main.hs). can you investigate, please?